src/editor/components/modals/reversion-modal.jsx

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import PropTypes from "prop-types";
+import Modal from "../../../shared/components/modal";
+import ModalTitle from "../../../shared/components/modal-title";
+import ModalContent from "../../../shared/components/modal-content";
+import ModalCall from "../../../shared/components/modal-call";
+import { TextButton } from "../../../shared/components/buttons";
+
+export default class DeleteModal extends React.Component {
```

Comment:
DeleteModal is clearly wrong -- maybe use something like `RevertConfirmationModal`? (best to give the file a similar name)